ui/raidboss/data/06-ew/trial/endsinger.ts

```diff
@@ -0,0 +1,251 @@
+      netRegex: NetRegexes.startsUsing({ id: '6652', source: 'The Endsinger', capture: true }),
+      response: Responses.tankCleave(),
+    },
+    {
+      id: 'Endsinger Epigonoi',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6646', source: 'The Endsinger', capture: true }),
+      condition: (_data, matches) => {
+        return matches.x !== '100' && matches.y !== '100';
+      },
+      suppressSeconds: 3,
+      infoText: (_data, matches, output) => {
+        if (matches.x === '100' || matches.y === '100')
+          return output.intercardinal?.();
+
+        return output.cardinal?.();
+      },
+      outputStrings: {
+        cardinal: {
+          en: 'Cardinal edge',
+        },
+        intercardinal: {
+          en: 'Intercardinal edge',
+        },
+      },
+    },
+    {
+      id: 'Endsinger Interstellar',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '664D', source: 'The Endsinger', capture: true }),
+      alertText: (_data, matches, output) => {
+        const x = parseFloat(matches.x);
+        const y = parseFloat(matches.y);
+
+        // Handle cardinals the easy way
+        if (x === 100) {
+          return output.direction!({
+            dir1: output.east!(),
+            dir2: output.west!(),
+          });
+        }
+        if (y === 100) {
+          return output.direction!({
+            dir1: output.north!(),
+            dir2: output.south!(),
+          });
+        }
+
+        if (x < 100) {
+          if (y < 100) {
+            return output.direction!({
+              dir1: output.northeast!(),
+              dir2: output.southwest!(),
+            });
+          }
+          return output.direction!({
+            dir1: output.northwest!(),
+            dir2: output.southeast!(),
+          });
+        }
+        if (y < 100) {
+          return output.direction!({
+            dir1: output.northwest!(),
+            dir2: output.southeast!(),
+          });
+        }
+        return output.direction!({
+          dir1: output.northeast!(),
+          dir2: output.southwest!(),
+        });
+      },
+      outputStrings: {
+        north: Outputs.north,
+        northeast: Outputs.northeast,
+        east: Outputs.east,
+        southeast: Outputs.southeast,
+        south: Outputs.south,
+        southwest: Outputs.southwest,
+        west: Outputs.west,
+        northwest: Outputs.northwest,
+        direction: {
+          en: '${dir1} / ${dir2}',
+        },
+      },
+    },
+    {
+      id: 'Endsinger Nemesis',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '664E', source: 'The Endsinger', capture: true }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Endsinger Ultimate Fate',
+      type: 'GameLog',
+      netRegex: NetRegexes.gameLog({ code: '0039', line: '4 seconds until oblivion is sealed\\.\\.\\.' }),
+      suppressSeconds: 65,
```

Comment:
Why is there a suppress?

Reply:
The line is repeated for the recast of the ability, which can't be tank LB'd, so there's no point in having the trigger fire again.